include/at_params.h

```diff
@@ -0,0 +1,221 @@
+/**@file at_params.h
+ *
+ * @brief Store a list of AT commands/responses parameters.
+ *
+ * A parameter list contains an array of parameters defined by a type
+ * and a value. Those parameters could be arguments of an AT command for
+ * instance. They can be numeric or string values.
+ * The same list of parameters can be reused. Each parameter can be
+ * updated or cleared. Once the parameter list is created, its size
+ * cannot be changed. All parameters values are copied in the list.
+ * Parameters should be cleared to free that memory. Getter and setter
+ * methods are available to read parameter values.
+ *
+ */
+
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef AT_PARAMS_H_
+#define AT_PARAMS_H_
+
+#include <stdint.h>
```

Comment:
`u32_t`, `u16_t` etc are not defined in `stdint.h`. Include `zephyr/types.h` instead.

Reply:
Fixed